projects/crates.io/git-delta/package.yml

```diff
@@ -0,0 +1,24 @@
+distributable:
+  url: https://github.com/dandavison/delta/archive/refs/tags/{{version}}.tar.gz
+  strip-components: 1
+
+provides:
+  - bin/delta
+
+versions:
+  github: dandavison/delta/tags
+
+dependencies:
+  darwin:
+    zlib.net: ^1
```

Comment:
it only needs zlib or darwin? This seems unlikely.

Reply:
In my testing zlib was only needed on darwin.